lib/s3routes/routes/routeGET.js

```diff
@@ -103,16 +103,16 @@ function routerGET(request, response, api, log, statsClient, dataRetrievalFn) {
                     return routesUtils.responseXMLBody(err, xml, response, log,
                         corsHeaders);
                 });
-        } else if (request.query.tagging !== undefined) {
-            // GET object Tagging
-            api.callApiMethod('objectGetTagging', request, response, log,
+        } else if (request.query['legal-hold'] !== undefined) {
+            // GET object legal hold
```

Comment:
this comment is probably not required given the if condition and call api method name handle this already